src/Illuminate/Testing/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "illuminate/testing",
+    "description": "The Illuminate Testing package.",
+    "license": "MIT",
+    "homepage": "https://laravel.com",
+    "support": {
+        "issues": "https://github.com/laravel/framework/issues",
+        "source": "https://github.com/laravel/framework"
+    },
+    "authors": [
+        {
+            "name": "Taylor Otwell",
+            "email": "taylor@laravel.com"
+        }
+    ],
+    "require": {
+        "php": "^7.2.5",
+        "mockery/mockery": "^1.3.1",
```

Comment:
Needs to also depend on `illuminate/console`, or at least suggest it, due to https://github.com/crynobone/framework/blob/testing-component/src/Illuminate/Testing/PendingCommand.php#L162-L164.

Also, need to depend on the illuminate/contracts repo?

Reply:
Also need illuminate/support for all the TestResponse, and should "suggest" illuminate/database.

Reply:
I would agree we need to require `illuminate/contracts` and `illuminate/support`.

But would be based if `illuminate/console` (for `PendingCommand`), `illuminate/http` (for `TestResponse`) and `illuminate/database` (for database assertions) should just be suggest.

Reply:
For example you might want to use this package to test `illuminate/database` running outside of Laravel or Lumen (using capsule).

I not sure how useful PendingCommand or TestResponse on similar usage right now.

Reply:
👍